scala/scalafmt/phase_scalafmt_ext.bzl

```diff
@@ -0,0 +1,54 @@
+load(
```

Comment:
why does this file's name ends with `_ext`?

Reply:
It means `extension`. I think we need to decide a naming convention of this kind of files before it gets too many.

Reply:
How do you define an extension? I'm guessing it depends on how you define the other thing (core?)? Maybe it should just be phase_scalafmt?
I'm not against but I have a hunch there's a methodology/pattern here which is clear to you and which I'm trying to make explicit so we can discuss and agree

Reply:
Let’s close this one for now and defer it to the convention discussion.
I think this question of whether we should separate to two files should also be there